package.json

```diff
@@ -88,7 +89,11 @@
   },
   "size-limit": [
     {
-      "path": "dist/index.js",
+      "path": "./dist/index.cjs",
+      "limit": "6 kB"
+    },
+    {
+      "path": "./dist/index.module.js",
```

Comment:
These new tests reflect the new output files (`index.cjs` and `index.module.js` which replace the old output/built file `index.js` by setting `type` to `module`).